components/definitions/definitions.js

```diff
@@ -0,0 +1,127 @@
+import domready from '../../assets/js/domready';
+
+export const classAnchor = 'definition__anchor';
+export const classDialog = 'dialog';
+export const classDialogButton = 'dialog__button';
+export const classDialogDocument = 'dialog__document';
+
+export const idDialogMessage = 'd-message';
+
+export const attrResponse = 'data-dialog-response';
+export const attrDataDialogCall = 'data-dialog-call';
+export const attrDataTitle = 'data-title';
+
+export default function definitionDialog() {
+  const anchors = document.getElementsByClassName(classAnchor);
+  for (let anchor of anchors) {
+    // Replace <a> tag with <button> tag.
+    anchor.outerHTML = anchor.outerHTML
+      .replace(/<a/g, '<button')
+      .replace(/<\/a>/g, '</button>');
+  }
+  const buttons = document.getElementsByClassName(classAnchor);
+  for (let button of buttons) {
+    button.onclick = function(e) {
+      e.preventDefault();
+      // define the dialog element
+      let dialog = document.getElementsByClassName(classDialog)[0];
+
+      // record the trigger element
+      let trigger = button.getAttribute('id')
+        ? button.getAttribute('id')
+        : 'trigger';
+
+      // open dialog and add roles
+      dialog.setAttribute('tabindex', '0');
+      dialog.setAttribute('open', 'true');
+      dialog.setAttribute('role', 'alertdialog');
+      dialog.setAttribute('aria-labelledby', idDialogMessage);
+
+      // retrieve custom close button wording, if any
+      let closeText = button.getAttribute(attrResponse)
+        ? button.getAttribute(attrResponse)
+        : 'close';
+
+      // build the dialog markup
+      dialog.insertAdjacentHTML(
+        'beforeend',
+        '<div class="dialog__wrapper">' +
+          '<div class="container">' +
+            '<div class="grid grid--reverse">' +
+              '<div class="grid__col col-4@m">' +
```

Comment:
Sorry - this requires the `pull-1@m` added to it so that it is placed over the ‘gutter’ column between the main body col7 and the col4 containing the progress indicator